### PR TITLE
Added clickhouse support

### PR DESCRIPTION
**Pull request by ilaktasic**
- added support for clickhouse testcontainers 
- custom properties path support

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -35,9 +35,19 @@ Usual use cases include:
...
+    * [ClickHouse](#ClickHouse)
+        * [Tests](#ClickHouseTests)
+        * [Local development](#ClickHouseLocalDevelopment)
+        * [Docker image version](#ClickHouseDockerImageVersion)
+        
 
 ## <a name="News"></a> News
 
+### 3.2.0
+####ClickHouse:
```
Comment (lpandzic) [inline README.md]: formatting is broken without space before heading - check rendered MD under https://github.com/infobip/infobip-testcontainers-spring-boot-starter after selecting branch.

File: README.md [hunk 2]
```diff
@@ -451,6 +461,72 @@ To change the docker image used simply add the following property (e.g. in yaml)
 ```yaml
 testcontainers.rabbit.docker.image: rabbitmq:3.6.14-alpine
 ```
+<a name="ClickHouse"></a>
+### ClickHouse
+<a name="ClickHouseTests"></a>
+##Tests
```
Comment (lpandzic) [inline README.md]: multiple examples like this

File: infobip-clickhouse-testcontainers-spring-boot-starter/pom.xml [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+			<version>${testcontainers.version}</version>
+		</dependency>
+		<dependency>
+			<groupId>ru.yandex.clickhouse</groupId>
+			<artifactId>clickhouse-jdbc</artifactId>
+			<version>${clickhouse-jdbc.version}</version>
+		</dependency>
+		<dependency>
+			<groupId>org.springframework</groupId>
+			<artifactId>spring-jdbc</artifactId>
```
Comment (lpandzic) [inline infobip-clickhouse-testcontainers-spring-boot-starter/pom.xml]: spring-boot-starter-jdbc doesn't contain this already?

File: README.md (hunk 1 above)
Comment (ilaktasic) [reply README.md]: fixed, in Idea formatting was fine

File: infobip-clickhouse-testcontainers-spring-boot-starter/pom.xml (hunk 3 above)
Comment (ilaktasic) [reply infobip-clickhouse-testcontainers-spring-boot-starter/pom.xml]: removed

File: README.md (hunk 2 above)
Comment (ilaktasic) [reply README.md]: fixed

File: README.md [hunk 4]
```diff
@@ -35,9 +35,20 @@ Usual use cases include:
         * [Tests](#RabbitMqTests)
         * [Local development](#RabbitMqLocalDevelopment)
         * [Docker image](#RabbitMqDockerImage)
+    * [ClickHouse](#ClickHouse)
```
Comment (lpandzic) [inline README.md:38]: Link doesn't work, it jumps to news section.
Probably best to move news to changelog like it's done in querydsl here: https://github.com/infobip/infobip-spring-data-querydsl#Changelog

Comment (ilaktasic) [reply README.md:38]: yup doesn't work also for MSSQL, but usage is right below change log so issue is not that visible

Comment (ilaktasic) [reply README.md:38]: changelog moved

**Review by lpandzic — APPROVED** [review]

**Pull request merged**